### PR TITLE
Run Prettier on HTML code fences, part 4

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to #20638

The PR focuses only on HTML code fences.

Idea is to gradually prettify all the CSS code fences before the full automation.

### CODE REVIEW

File: files/en-us/learn/css/css_layout/positioning/index.md [hunk 1]
```diff
@@ -384,13 +450,26 @@ You should now see the finished example:
...
-<p>We are separated by our margins. Because of margin collapsing, we are separated by the width of one of our margins, not both.</p>
-
-<p>inline elements <span>like this one</span> and <span>this one</span> sit on the same line as one another, and adjacent text nodes, if there is space on the same line. Overflowing inline elements <span>wrap onto a new line if possible — like this one containing text</span>, or just go on to a new line if not, much like this image will do: <img src="long.jpg" alt="snippet of cloth"></p>
+<p>
+  We are separated by our margins. Because of margin collapsing, we are
+  separated by the width of one of our margins, not both.
+</p>
+
+<p>
+  inline elements <span>like this one</span> and <span>this one</span> sit on
```
Comment (teoli2003) [inline files/en-us/learn/css/css_layout/positioning/index.md:466]:
```suggestion
  Inline elements <span>like this one</span> and <span>this one</span> sit on
```

File: files/en-us/learn/forms/how_to_structure_a_web_form/index.md [hunk 2]
```diff
@@ -195,73 +195,75 @@ Let's put these ideas into practice and build a slightly more involved form —
...
 
 4. Inside the `<form>` tags, add a heading and paragraph to inform users how required fields are marked:
 
    ```html
    <h1>Payment form</h1>
-   <p>Required fields are followed by <strong><span aria-label="required">*</span></strong>.</p>
+   <p>
+     Required fields are followed by
+     <strong><span aria-label="required">*</span></strong
+     >.
```
Comment (teoli2003) [inline files/en-us/learn/forms/how_to_structure_a_web_form/index.md:214]:
```suggestion
     <strong><span aria-label="required">*</span></strong>.
```

File: files/en-us/learn/html/introduction_to_html/advanced_text_formatting/index.md [hunk 3]
```diff
@@ -113,11 +133,12 @@ Eggs
 The glue that binds the cake together.
 Coffee
 The drink that gets the world running in the morning.
-A light brown color.</textarea>
+A light brown color.</textarea
+>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/advanced_text_formatting/index.md:137]:
```suggestion
A light brown color.
</textarea>
```

File: files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md [hunk 4]
```diff
@@ -82,15 +82,15 @@ We've already seen the {{htmlelement("title")}} element in action — this can b
 
 2. Now open the file in your browser. You should see something like this:
 
-   ![A simple web page with the title set to \<title> element, and the \<h1> set to \<h1> element.](title-example.png)
+   ![A simple web page with the title set to <title> element, and the <h1> set to <h1> element.](title-example.png)
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:85]:
```suggestion
   ![A simple web page with the title set to \<title> element, and the \<h1> set to \<h1> element.](title-example.png)
```

File: files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md [hunk 5]
```diff
@@ -82,15 +82,15 @@ We've already seen the {{htmlelement("title")}} element in action — this can b
...
 
    It should now be completely obvious where the `<h1>` content appears and where the `<title>` content appears!
 
 3. You should also try opening the code up in your code editor, editing the contents of these elements, then refreshing the page in your browser. Have some fun with it.
 
 The `<title>` element contents are also used in other ways. For example, if you try bookmarking the page (_Bookmarks > Bookmark This Page_ or the star icon in the URL bar in Firefox), you will see the `<title>` contents filled in as the suggested bookmark name.
 
-![A webpage being bookmarked in Firefox; the bookmark name has been automatically filled in with the contents of the \<title> element ](bookmark-example.png)
+![A webpage being bookmarked in Firefox; the bookmark name has been automatically filled in with the contents of the <title> element ](bookmark-example.png)
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:93]:
```suggestion
![A webpage being bookmarked in Firefox; the bookmark name has been automatically filled in with the contents of the \<title> element ](bookmark-example.png)
```

File: files/en-us/learn/forms/how_to_build_custom_form_controls/example_2/index.md [hunk 6]
```diff
@@ -32,7 +32,8 @@ This is the second example that explain [how to build custom form widgets](/en-U
       <li class="option">Apple</li>
     </ul>
   </div>
-<form>
+  <form></form>
```
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply files/en-us/learn/forms/how_to_build_custom_form_controls/example_2/index.md:35]: the original code didn’t have `/` in closing tag

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 7]
```diff
@@ -687,8 +696,7 @@ In the examples above, you may have noticed that a lot of whitespace is included
 ```html
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:696]:
```suggestion
```html-nolint
```

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 8]
```diff
@@ -684,11 +693,10 @@ textarea.onkeyup = () => {
 
 In the examples above, you may have noticed that a lot of whitespace is included in the code. This is optional. These two code snippets are equivalent:
 
-```html
+```html-nolint
 <p>Dogs are silly.</p>
 
-<p>Dogs        are
-         silly.</p>
+<p>Dogs are silly.</p>
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:699]:
```suggestion
<p>Dogs         are
       silly.</p>
```

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 9]
```diff
@@ -684,11 +693,11 @@ textarea.onkeyup = () => {
 
 In the examples above, you may have noticed that a lot of whitespace is included in the code. This is optional. These two code snippets are equivalent:
 
-```html
+```html-nolint
 <p>Dogs are silly.</p>
 
-<p>Dogs        are
-         silly.</p>
+<p>Dogs         are
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:699]:
```suggestion
<p>Dogs        are
```

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 10]
```diff
@@ -684,11 +693,11 @@ textarea.onkeyup = () => {
...
+       silly.</p>
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:700]:
```suggestion
         silly.</p>
```

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 11]
```diff
@@ -714,7 +723,8 @@ The character reference equivalent could be easily remembered because the text i
 In the example below, there are two paragraphs:
 
 ```html
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:725]:
```suggestion
```html-nolint
```

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 12]
```diff
@@ -714,7 +723,8 @@ The character reference equivalent could be easily remembered because the text i
...
-<p>In HTML, you define a paragraph using the <p> element.</p>
+<p>In HTML, you define a paragraph using the</p>
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:726]:
```suggestion
<p>In HTML, you define a paragraph using the </p> element.</p>
```

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md [hunk 13]
```diff
@@ -279,10 +289,7 @@ Every unordered list starts off with a {{htmlelement("ul")}} element—this wrap
 
 ```html
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md:290]:
```suggestion
```html-nolint
```

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md [hunk 14]
```diff
@@ -279,10 +289,7 @@ Every unordered list starts off with a {{htmlelement("ul")}} element—this wrap
...
 <ul>
-milk
-eggs
-bread
-hummus
+  milk eggs bread hummus
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md:292]:
```suggestion
- milk
- eggs
- bread
- hummus
```

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 15]
```diff
@@ -687,8 +696,7 @@ In the examples above, you may have noticed that a lot of whitespace is included
 ```html
 <p>Dogs are silly.</p>
 
-<p>Dogs        are
-         silly.</p>
+<p>Dogs are silly.</p>
```
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply files/en-us/learn/html/introduction_to_html/getting_started/index.md]: marked the code fence with `html-nolint`

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md]: `html-nolint`

**Comment by OnkarRuikar** [comment]
Thanks @rubiesonthesky!

Those form tags weren’t closed properly in the original code.
I’ve marked other no format code fences with `html-nolint`

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 16]
```diff
@@ -684,7 +693,7 @@ textarea.onkeyup = () => {
 
 In the examples above, you may have noticed that a lot of whitespace is included in the code. This is optional. These two code snippets are equivalent:
 
-```html
+```html-nolint
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:696]:
```suggestion
```html
```

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 17]
```diff
@@ -713,8 +722,9 @@ The character reference equivalent could be easily remembered because the text i
 
 In the example below, there are two paragraphs:
 
-```html
-<p>In HTML, you define a paragraph using the <p> element.</p>
+```html-nolint
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:725]:
```suggestion
```html
```

File: files/en-us/learn/html/introduction_to_html/advanced_text_formatting/index.md [hunk 18]
```diff
@@ -706,7 +765,9 @@ The basic example above just provides a simple machine readable date, but there
 <!-- Date and time -->
 <time datetime="2016-01-20T19:30">7.30pm, 20 January 2016</time>
 <!-- Date and time with timezone offset -->
-<time datetime="2016-01-20T19:30+01:00">7.30pm, 20 January 2016 is 8.30pm in France</time>
+<time datetime="2016-01-20T19:30+01:00"
+  >7.30pm, 20 January 2016 is 8.30pm in France</time
+>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/advanced_text_formatting/index.md:770]:
```suggestion
<time datetime="2016-01-20T19:30+01:00"
  >7.30pm, 20 January 2016 is 8.30pm in France</time
>
```
```suggestion
<time datetime="2016-01-20T19:30+01:00">
  7.30pm, 20 January 2016 is 8.30pm in France
</time>
```

File: files/en-us/learn/html/introduction_to_html/creating_hyperlinks/index.md [hunk 19]
```diff
@@ -81,10 +82,14 @@ Another attribute you may want to add to your links is `title`.
...
-   title="The best place to find more information about Mozilla's
-          mission and how to contribute">the Mozilla homepage</a>.
+<p>
+  I'm creating a link to
+  <a
+    href="https://www.mozilla.org/en-US/"
+    title="The best place to find more information about Mozilla's
+          mission and how to contribute"
+    >the Mozilla homepage</a
+  >.
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/creating_hyperlinks/index.md:92]:
```suggestion
  <a
    href="https://www.mozilla.org/en-US/"
    title="The best place to find more information about Mozilla's
          mission and how to contribute"
    >
    the Mozilla homepage
  </a>.
```

File: files/en-us/learn/html/introduction_to_html/debugging_html/index.md [hunk 20]
```diff
@@ -110,7 +110,8 @@ It's time to study the permissive nature of HTML code.
    - The incorrect nesting has been fixed by the browser as shown here:
 
      ```html
-     <strong>strong
+     <strong
+       >strong
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/debugging_html/index.md:114]:
```suggestion
     <strong>
       strong
```

File: files/en-us/learn/html/introduction_to_html/debugging_html/index.md [hunk 21]
```diff
@@ -120,8 +121,10 @@ It's time to study the permissive nature of HTML code.
 
      ```html
      <li>
-       <strong>Unclosed attributes: Another common source of HTML problems.
-       Let's look at an example: </strong>
+       <strong
+         >Unclosed attributes: Another common source of HTML problems. Let's
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/debugging_html/index.md:125]:
```suggestion
       <strong>
         Unclosed attributes: Another common source of HTML problems. Let's
```

File: files/en-us/learn/html/introduction_to_html/document_and_website_structure/index.md [hunk 22]
```diff
@@ -192,9 +215,14 @@ Sometimes you'll come across a situation where you can't find an ideal semantic
...
-<p>The King walked drunkenly back to his room at 01:00, the beer doing nothing to aid
-him as he staggered through the door <span class="editor-note">[Editor's note: At this point in the
-play, the lights should be down low]</span>.</p>
+<p>
+  The King walked drunkenly back to his room at 01:00, the beer doing nothing to
+  aid him as he staggered through the door
+  <span class="editor-note"
+    >[Editor's note: At this point in the play, the lights should be down
+    low]</span
+  >.
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/document_and_website_structure/index.md:224]:
```suggestion
  <span class="editor-note">
    [Editor's note: At this point in the play, the lights should be down low]
  </span>.
```

File: files/en-us/learn/html/introduction_to_html/document_and_website_structure/index.md [hunk 23]
```diff
@@ -206,12 +234,13 @@ In this case, the editor's note is supposed to merely provide extra direction fo
...
   <ul>
     <li>
-      <p><a href=""><strong>Silver earrings</strong></a>: $99.95.</p>
-      <img src="../products/3333-0985/thumb.png" alt="Silver earrings">
-    </li>
-    <li>
-      …
+      <p>
+        <a href=""><strong>Silver earrings</strong></a
+        >: $99.95.
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/document_and_website_structure/index.md:239]:
```suggestion
        <a href=""><strong>Silver earrings</strong></a>:
        $99.95.
```

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 24]
```diff
@@ -478,7 +483,9 @@ Make sure you don't mix single quotes and double quotes. This example (below) sh
 However, if you use one type of quote, you can include the other type of quote _inside_ your attribute values:
 
 ```html
-<a href="https://www.example.com" title="Isn't this fun?">A link to my example.</a>
+<a href="https://www.example.com" title="Isn't this fun?"
+  >A link to my example.</a
+>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:488]:
```suggestion
<a href="https://www.example.com" title="Isn't this fun?">
  A link to my example.
</a>
```

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 25]
```diff
@@ -490,7 +497,9 @@ To use quote marks inside other quote marks of the same type (single quote or do
 Instead, you need to do this:
 
 ```html
-<a href='https://www.example.com' title='Isn&apos;t this fun?'>A link to my example.</a>
+<a href="https://www.example.com" title="Isn't this fun?"
+  >A link to my example.</a
+>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:502]:
```suggestion
<a href="https://www.example.com" title="Isn't this fun?">
  A link to my example.
</a>
```

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md (hunk 11 above)
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:725]:
```suggestion
```html-nolint
```

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 26]
```diff
@@ -714,7 +723,8 @@ The character reference equivalent could be easily remembered because the text i
 In the example below, there are two paragraphs:
 
 ```html
-<p>In HTML, you define a paragraph using the <p> element.</p>
+<p>In HTML, you define a paragraph using the </p> element.</p>
+<p>element.</p>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:727]:
```suggestion
```

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md [hunk 27]
```diff
@@ -115,19 +120,22 @@ If you make a mistake, you can always reset it using the _Reset_ button. If you
...
+  Press Esc to move focus away from the code area (Tab inserts a tab character).
+</p>
 
-<textarea id="code" class="input" style="min-height: 100px; width: 95%">My short story I am a statistician and my name is Trish.
+<textarea id="code" class="input" style="min-height: 100px; width: 95%">
+My short story I am a statistician and my name is Trish.
 
-My legs are made of cardboard and I am married to a fish.</textarea>
+My legs are made of cardboard and I am married to a fish.</textarea
+>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md:134]:
```suggestion
My legs are made of cardboard and I am married to a fish.
</textarea>
```

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md [hunk 28]
```diff
@@ -277,12 +287,12 @@ hummus
 
 Every unordered list starts off with a {{htmlelement("ul")}} element—this wraps around all the list items:
 
-```html
+```html-nolint
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md:290]:
```suggestion
```html
```

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md [hunk 29]
```diff
@@ -304,20 +314,23 @@ Try editing the live sample below to create your very own HTML unordered list.
...
+</p>
 
-<textarea id="code" class="input" style="min-height: 100px; width: 95%">milk
+<textarea id="code" class="input" style="min-height: 100px; width: 95%">
+milk
 eggs
 bread
-hummus</textarea>
+hummus</textarea
+>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md:329]:
```suggestion
hummus
</textarea>
```

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md [hunk 30]
```diff
@@ -455,21 +468,24 @@ Try editing the live sample below to create your very own HTML ordered list.
...
 
-<textarea id="code" class="input" style="min-height: 200px; width: 95%">Drive to the end of the road
+<textarea id="code" class="input" style="min-height: 200px; width: 95%">
+Drive to the end of the road
 Turn right
 Go straight across the first two roundabouts
 Turn left at the third roundabout
-The school is on your right, 300 meters up the road</textarea>
+The school is on your right, 300 meters up the road</textarea
+>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md:484]:
```suggestion
The school is on your right, 300 meters up the road
</textarea>
```

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md [hunk 31]
```diff
@@ -620,11 +638,12 @@ So at this point in the article, you have all the information you need to mark u
 
   Refrigerate the finished hummus in a sealed container. You should be able to use it for about a week after you've made it. If it starts to become fizzy, you should definitely discard it.
 
-  Hummus is suitable for freezing; you should thaw it and use it within a couple of months.</textarea>
+  Hummus is suitable for freezing; you should thaw it and use it within a couple of months.</textarea
+>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md:642]:
```suggestion
  Hummus is suitable for freezing; you should thaw it and use it within a couple of months.
</textarea>
```

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md [hunk 32]
```diff
@@ -816,24 +841,27 @@ In this active learning section, we've provided an editable example. Inside it,
...
+<h1>Important notice</h1>
 <p>On Sunday January 9th 2010, a gang of goths were
   spotted stealing several garden gnomes from a
   shopping center in downtown Milwaukee. They were
   all wearing green jumpsuits and silly hats, and
   seemed to be having a whale of a time. If anyone
    has any information about this incident, please
-    contact the police now.</p></textarea>
+    contact the police now.</p></textarea
+>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md:860]:
```suggestion
  contact the police now.</p>
</textarea>
```

File: files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md [hunk 33]
```diff
@@ -172,11 +176,15 @@ As you travel around the web, you'll find other types of metadata, too. A lot of
...
+<meta
+  property="og:image"
+  content="https://developer.mozilla.org/static/img/opengraph-logo.png" />
+<meta
+  property="og:description"
+  content="The Mozilla Developer Network (MDN) provides
 information about Open Web technologies including HTML, CSS, and APIs for both Web sites
-and HTML Apps. It also documents Mozilla products, like Firefox OS.">
-<meta property="og:title" content="Mozilla Developer Network">
+and HTML Apps. It also documents Mozilla products, like Firefox OS." />
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:186]:
```suggestion
and HTML Apps." />
```

File: files/en-us/learn/javascript/asynchronous/introducing_workers/index.md [hunk 34]
```diff
@@ -104,27 +104,26 @@ The "index.html" file and the "style.css" files are already complete:
...
-    <input type="text" id="quota" name="quota" value="1000000">
+    <input type="text" id="quota" name="quota" value="1000000" />
 
     <button id="generate">Generate primes</button>
     <button id="reload">Reload</button>
 
-    <textarea id="user-input" rows="5" cols="62">Try typing in here immediately after pressing "Generate primes"</textarea>
+    <textarea id="user-input" rows="5" cols="62">
+Try typing in here immediately after pressing "Generate primes"</textarea
+    >
```
Comment (teoli2003) [inline files/en-us/learn/javascript/asynchronous/introducing_workers/index.md:123]:
```suggestion
Try typing in here immediately after pressing "Generate primes"
</textarea>
```

File: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/ember_interactivity_events_state/index.md [hunk 35]
```diff
@@ -70,13 +70,8 @@ We can capture the [`keydown`](/en-US/docs/Web/API/Document/keydown_event) event
 Add the new line shown below to your `header.hbs` file:
 
 ```html
```
Comment (teoli2003) [inline files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/ember_interactivity_events_state/index.md:72]:
```suggestion
```html-nolint
```

File: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/ember_interactivity_events_state/index.md [hunk 36]
```diff
@@ -70,13 +70,8 @@ We can capture the [`keydown`](/en-US/docs/Web/API/Document/keydown_event) event
...
-<input
-  class='new-todo'
-  aria-label='What needs to be done?'
-  placeholder='What needs to be done?'
-  autofocus
-  \{{on 'keydown' this.onKeyDown}}
->
+<input class='new-todo' aria-label='What needs to be done?' placeholder='What
+needs to be done?' autofocus \{{on 'keydown' this.onKeyDown}} >
```
Comment (teoli2003) [inline files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/ember_interactivity_events_state/index.md:74]:
```suggestion
<input
  class='new-todo'
  aria-label='What needs to be done?'
  placeholder='What needs to be done?'
  autofocus
  \{{on 'keydown' this.onKeyDown}}
>
```

File: files/en-us/web/accessibility/aria/annotations/index.md [hunk 37]
```diff
@@ -113,9 +129,21 @@ If the descriptive text does appear in the UI (it should for this example), you
...
+        <li>
+          <label
+            >Orange: <input type="radio" name="fruit" value="orange"
+          /></label>
+        </li>
+        <li>
+          <label
+            >Banana: <input type="radio" name="fruit" value="banana"
+          /></label>
+        </li>
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/annotations/index.md:146]:
```suggestion
        <li>
          <label>
            Apple: <input type="radio" name="fruit" value="apple" />
          </label>
        </li>
        <li>
          <label>
            Orange: <input type="radio" name="fruit" value="orange" />
          </label>
        </li>
        <li>
          <label>
            Banana: <input type="radio" name="fruit" value="banana" />
          </label>
        </li>
```

File: files/en-us/web/accessibility/aria/annotations/index.md [hunk 38]
```diff
@@ -127,31 +155,48 @@ If the descriptive text does appear in the UI (it should for this example), you
 A common wish in online document systems like Google Docs is to be able to track changes, to see what reviewers or editors have suggested as changes to the text, before the managing editor or author accepts or rejects those changes. The semantics for this have long been available in HTML, via the {{HTMLElement('ins')}} and {{HTMLElement('del')}} elements:
 
 ```html
-<p>Freida's pet is a <del>black Cat called Luna</del><ins>purple Tyrannosaurus Rex called Tiny</ins>.</p>
+<p>
+  Freida's pet is a <del>black Cat called Luna</del
+  ><ins>purple Tyrannosaurus Rex called Tiny</ins>.
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/annotations/index.md:160]:
```suggestion
  Freida's pet is a <del>black Cat called Luna</del>
  <ins>purple Tyrannosaurus Rex called Tiny</ins>.
```

File: files/en-us/web/accessibility/aria/annotations/index.md [hunk 39]
```diff
@@ -127,31 +155,48 @@ If the descriptive text does appear in the UI (it should for this example), you
...
+</p>
 ```
 
 With the new additions, you now have new roles available to provide the same semantics, should you be unable to use {{HTMLElement('ins')}} and {{HTMLElement('del')}} elements for some reason:
 
 ```html
-<p>Freida's pet is a <span role="deletion">black Cat called Luna</span><span role="insertion">purple Tyrannosaurus Rex called Tiny</span>.</p>
+<p>
+  Freida's pet is a <span role="deletion">black Cat called Luna</span
+  ><span role="insertion">purple Tyrannosaurus Rex called Tiny</span>.
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/annotations/index.md:169]:
```suggestion
  Freida's pet is a <span role="deletion">black Cat called Luna</span>
  <span role="insertion">purple Tyrannosaurus Rex called Tiny</span>.
```

File: files/en-us/web/accessibility/aria/annotations/index.md [hunk 40]
```diff
@@ -127,31 +155,48 @@ If the descriptive text does appear in the UI (it should for this example), you
...
 
 ```html
-<p>Freida's pet is a
-  <span role="suggestion"><span role="deletion">black Cat called Luna</span><span role="insertion">purple Tyrannosaurus Rex called Tiny</span></span>.
+<p>
+  Freida's pet is a
+  <span role="suggestion"
+    ><span role="deletion">black Cat called Luna</span
+    ><span role="insertion">purple Tyrannosaurus Rex called Tiny</span></span
+  >.
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/annotations/index.md:181]:
```suggestion
  <span role="suggestion">
    <span role="deletion">black Cat called Luna</span>
    <span role="insertion">purple Tyrannosaurus Rex called Tiny</span>
  </span
  >.
```

File: files/en-us/web/accessibility/aria/annotations/index.md [hunk 41]
```diff
@@ -127,31 +155,48 @@ If the descriptive text does appear in the UI (it should for this example), you
...
 
 ```html
-<p>Freida's pet is a
-  <span role="suggestion" aria-details="comment-source"><span role="deletion">black Cat called Luna</span><span role="insertion">purple Tyrannosaurus Rex called Tiny</span></span>.
+<p>
+  Freida's pet is a
+  <span role="suggestion" aria-details="comment-source"
+    ><span role="deletion">black Cat called Luna</span
+    ><span role="insertion">purple Tyrannosaurus Rex called Tiny</span></span
+  >.
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/annotations/index.md:193]:
```suggestion
  <span role="suggestion" aria-details="comment-source">
    <span role="deletion">black Cat called Luna</span>
    <span role="insertion">purple Tyrannosaurus Rex called Tiny</span>
  </span>.
```

**Comment by OnkarRuikar** [comment]
I've reverted all the changes in framework codes and removed `html-nolint` tangs.
Also fixed split <lable> tags.

**Comment by OnkarRuikar** [comment]
Sorry I didn't know your working on it at the same time.

File: files/en-us/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 42]
```diff
@@ -96,18 +96,18 @@ In this section, we'll look at the two problems illustrated above and show how t
...
 We can however use two attributes — {{htmlattrxref("srcset", "img")}} and {{htmlattrxref("sizes", "img")}} — to provide several additional source images along with hints to help the browser pick the right one. You can see an example of this in our [responsive.html](https://mdn.github.io/learning-area/html/multimedia-and-embedding/responsive-images/responsive.html) example on GitHub (see also [the source code](https://github.com/mdn/learning-area/blob/main/html/multimedia-and-embedding/responsive-images/responsive.html)):
 
 ```html
-<img srcset="elva-fairy-480w.jpg 480w,
-             elva-fairy-800w.jpg 800w"
-     sizes="(max-width: 600px) 480px,
+<img
+  srcset="elva-fairy-480w.jpg 480w, elva-fairy-800w.jpg 800w"
+  sizes="(max-width: 600px) 480px,
             800px"
```
Comment (teoli2003) [inline files/en-us/learn/html/multimedia_and_embedding/responsive_images/index.md:108]:
```suggestion
         800px"
```

File: files/en-us/learn/html/tables/basics/index.md [hunk 43]
```diff
@@ -60,137 +60,148 @@ The point of a table is that it is rigid. Information is easily interpreted by m
...
-   <td>-225</td>
-   <td>5</td>
-   <td>Declassified as a planet in 2006, but this <a href="https://www.usatoday.com/story/tech/2014/10/02/pluto-planet-solar-system/16578959/">remains controversial</a>.</td>
-  </tr>
- </tbody>
+  <caption>
+    Data about the planets of our solar system (Planetary facts taken from
+    <a href="https://nssdc.gsfc.nasa.gov/planetary/factsheet/"
+      >Nasa's Planetary Fact Sheet - Metric</a
+    >).
```
Comment (teoli2003) [inline files/en-us/learn/html/tables/basics/index.md]:
```suggestion
    <a href="https://nssdc.gsfc.nasa.gov/planetary/factsheet/">
      Nasa's Planetary Fact Sheet - Metric
    </a>).
```

File: files/en-us/learn/html/tables/basics/index.md [hunk 44]
```diff
@@ -60,137 +60,148 @@ The point of a table is that it is rigid. Information is easily interpreted by m
...
+      <td>153.3</td>
+      <td>5906.4</td>
+      <td>-225</td>
+      <td>5</td>
+      <td>
+        Declassified as a planet in 2006, but this
+        <a
+          href="https://www.usatoday.com/story/tech/2014/10/02/pluto-planet-solar-system/16578959/"
+          >remains controversial</a
+        >.
```
Comment (teoli2003) [inline files/en-us/learn/html/tables/basics/index.md]:
```suggestion
        <a
          href="https://www.usatoday.com/story/tech/2014/10/02/pluto-planet-solar-system/16578959/">
          remains controversial
        </a>.
```

File: files/en-us/learn/javascript/asynchronous/introducing_workers/index.md [hunk 45]
```diff
@@ -104,27 +104,25 @@ The "index.html" file and the "style.css" files are already complete:
...
     <label for="quota">Number of primes:</label>
-    <input type="text" id="quota" name="quota" value="1000000">
+    <input type="text" id="quota" name="quota" value="1000000" />
 
     <button id="generate">Generate primes</button>
     <button id="reload">Reload</button>
 
-    <textarea id="user-input" rows="5" cols="62">Try typing in here immediately after pressing "Generate primes"</textarea>
+    <textarea id="user-input" rows="5" cols="62">
+Try typing in here immediately after pressing "Generate primes"</textarea>
```
Comment (teoli2003) [inline files/en-us/learn/javascript/asynchronous/introducing_workers/index.md:122]:
```suggestion
Try typing in here immediately after pressing "Generate primes"
</textarea>
```

File: files/en-us/learn/javascript/building_blocks/conditionals/index.md [hunk 46]
```diff
@@ -690,11 +694,12 @@ select.addEventListener('change', () => {
 function update(bgColor, textColor) {
   html.style.backgroundColor = bgColor;
   html.style.color = textColor;
-}</textarea>
+}</textarea
+>
```
Comment (teoli2003) [inline files/en-us/learn/javascript/building_blocks/conditionals/index.md:698]:
```suggestion
}
</textarea>
```

**Review by teoli2003 — APPROVED** [review]

**Comment by OnkarRuikar** [comment]
Thanks!

**Pull request merged**